src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -506,4 +506,45 @@ public function frontendDevDisable($siteDir = 'default', array $opts = ['yes|y'
             ->remove($this->devServicesPath)
             ->run();
     }
+
+    /**
+     * Setup a site in GitHub Codespaces.
+     *
+     * @return \Robo\Result
+     *   The result of the set of tasks.
+     *
+     * @throws \Robo\Exception\TaskException
+     *
+     */
+    public function setupCodespaces(): Result
+    {
+        $this->io()->title('Symlinking file system.');
+        $docRootDir = Robo::config()->get('drupal_document_root') ?? 'web';
+        $codespaces_directory = getenv('PWD') . '/' . $docRootDir;
+        if (empty($codespaces_directory)) {
+            throw new TaskException($this, 'Codespaces directory is unavailable.');
+        }
+        $this->taskDeleteDir('/var/www/html')->run();
+        $result = $this->taskExec("ln -s $codespaces_directory /var/www/html")->run();
+
+        $this->io()->title('Start apache, forwarding port 80.');
+        $result = $this->taskExec('service apache2 start')->run();
+
+        $this->io()->title('Download database.');
+        $dbPath = $this->databaseDownload();
+        if (empty($dbPath)) {
+            throw new TaskException($this, 'Database download failed.');
+        }
+
+        $this->io()->section('Importing database.');
+        $result = $this->taskExec("zcat $dbPath | mysql -h db -u mariadb -pmariadb mariadb")->run();
+        $result = $this->taskExec('rm')->args($dbPath)->run();
+
+        $this->io()->section('Building theme.');
+        $result = $this->taskExec('composer robo theme:build')->run();
+
+        $this->io()->section('Clearing Drupal cache.');
+        $result = $this->taskExec('vendor/bin/drush cr')->run();
```

Comment:
Do we want to run just a `cr` or do we need a full `drush deploy` here?

Reply:
I think we do want to use `drush deploy`, since we're importing config twice in lando, we should probably do the same here as well to ensure we're fully up to date?